satellite/console/service.go

```diff
@@ -241,6 +257,49 @@ func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken st
 	return pp, nil
 }
 
+func (s *Service) AttachPaymentMethodToProject(ctx context.Context, paymentMethodID []byte, projectID uuid.UUID) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	authorization, err := GetAuth(ctx)
+	if err != nil {
+		return err
+	}
+
+	tx, err := s.store.BeginTx(ctx)
+	if err != nil {
+		return err
+	}
+
+	err = withTx(tx, func(tx DBTx) error {
```

Comment:
placing this callback into separate function will make this method more readable

Reply:
the same in the AddNewPaymentMethod method

Reply:
moreover these callbacks are pretty similar

Reply:
I second that!